examples/client/client.c

```diff
@@ -1281,6 +1281,9 @@ static void Usage(void)
 #endif
 #ifdef WOLFSSL_MULTICAST
     printf("%s", msg[++msgid]); /* -3 */
+#endif
+#ifdef HAVE_TRUSTED_CA
+    printf("-5          Use Trusted CA Key Indication\n");
```

Comment:
Can we also update for Japanese msg[] case?

Reply:
OK.